### PR TITLE
Manifest normalisation

### PR DESCRIPTION
**Pull request by guenhter**
Currently, only the pom entries are considered for normalisation.
This PR also normalizes the manifest-data.

Also allow writing better readable tests with a ProjectBuilder-DSL which allows
to create new `ProjectData` in a easy to read fashion. This removes the 
need for a lot of predefined Data-Fixtures.

### CODE REVIEW

File: src/test/groovy/com/github/jk1/license/filter/LicenseBundleNormalizerSpec.groovy [hunk 1]
```diff
@@ -1,264 +1,197 @@
...
-                filters = new LicenseBundleNormalizer()
-                renderer = new JsonReportRenderer()
-                configurations = ['forTesting']
-            }
-            dependencies {
-                forTesting "org.jetbrains:annotations:13.0"     // license-url: "http://www.apache.org/licenses/LICENSE-2.0.txt"
-                forTesting "io.netty:netty-common:4.1.17.Final" // license-url: "http://www.apache.org/licenses/LICENSE-2.0"
-            }
-        """
+    def "normalize the manifests license or to the appropriate bungle-license-name"() {
```
Comment (milesgould) [inline src/test/groovy/com/github/jk1/license/filter/LicenseBundleNormalizerSpec.groovy]: Typo: s/bungle/bundle

**Comment by jk1** [comment]
It seems like it conflicts with other pull requests merged recently. Would you mind rebasing it?

**Comment by guenhter** [comment]
Done.

**Comment by jk1** [comment]
Thanks a lot, merged.

**Pull request merged**